protocols/protocol/cloudstate/crdt.proto

```diff
@@ -328,43 +614,89 @@ message CrdtReply {
     bool streamed = 6;
 }
 
+//*CrdtStateAction* TODO
+//
+//---
+//
+//`Specification`:
+//
+//  - A `CrdtStateAction` is considered to be valid iff:
+//    - `action` MUST be one of the following:
+//      - `create` TODO
+//      - `update` TODO
+//      - `delete` TODO
+//    - `write_consistency` TODO
+//
+// FIXME set the ordinals to start at 1?
 message CrdtStateAction {
```

Comment:
Going through the CRDT implementation, I found we could perhaps describe more clearly how the proxy gets the initial state of a CRDT within the user library. Its written implicitly I think, put trying to implement the protocol by only the spec, I had to guess (lightly).

If I understand it correctly, the very first instance, the value of it, of a CRDT from a registered entity gets to know to the proxy by a CrdtStateAction being part of a CrdtReply in response to a CrdtStreamIn.Command as the proxy never can know of a CRDT before and with the first CrdtInit message.